### ISSUE TITLE
Build Size Too large

### ISSUE THREAD

**Issue by babitakapoor112** [shahen94/react-native-video-processing#233]
Apk file is becoming too large using this package. Without this package my apk file is 20 mb and after installing this package file size increases to 50 mb. Is there any solution for it?

**Comment by matinzd** [comment]
I have this problem too!!!

**Comment by noelsp** [comment]
Yes, i have the same problem.
![Captura de Pantalla 2020-05-06 a la(s) 11 46 14](https://user-images.githubusercontent.com/5714538/81191457-6c66e780-8f8f-11ea-967f-91bc02699795.png)

**Comment by namansukhwani** [comment]
I am having this same issue my apk size is 100 mb please help me if you have any solution

**Comment by gopisupreme** [comment]
> am having this same issue my apk size is 100 mb please help me if you have any solution

i am having same issue, Please help me

**Comment by namansukhwani** [comment]
> > am having this same issue my apk size is 100 mb please help me if you have any solution
> 
> i am having same issue, Please help me

I tried using proguard and multi patform build but that too was giving an error. Although the size of the library is 100mb.

**Comment by vuhoangminh199** [comment]
i am having same issue, Please help

**Comment by Karanbsankey** [comment]
> i am having same issue, Please help

any luck on reducing app size?

**Comment by angelop11** [comment]
I'm having this same issue, helppp

**Comment by vish2595** [comment]
I am having the same issue, Please help

**Comment by NareshBojja03** [comment]
Did any one have the update on this issue?
Highly appreciable if you have any solution @shahen94

**Comment by Seamus1989** [comment]
same issue, please help

**Comment by MrymZakani** [comment]
Same issue. 
The app size increased from 33M to 90M after using this library.

**Comment by matinzd** [comment]
> Same issue.
> The app size increased from 33M to 90M after using this library.

It will be better to use `ffmpeg` command on the native side for processing video.

**Comment by MrymZakani** [comment]
> > Same issue.
> > The app size increased from 33M to 90M after using this library.
> 
> It will be better to use `ffmpeg` command on the native side for processing video. You should write it on your own.

Thanks. May libraries such as react-native-ffmpeg work?

**Comment by matinzd** [comment]
> > > Same issue.
> > > The app size increased from 33M to 90M after using this library.
> > 
> > 
> > It will be better to use `ffmpeg` command on the native side for processing video. You should write it on your own.
> 
> Thanks. May libraries such as react-native-ffmpeg work?

Yes, I haven't seen that since now. I think it is a good libraray.

**Comment by aakanshu** [comment]
In logs I've seen due to missing Supported NDK version it is skipping compression of library. On installing latest ndk version (ie 23) it was failing with toolkit error which got removed from ndk version 19. Any idea if library compression can be done by have some compatible ndk version installed on android studio?

**Comment by datqb** [comment]
> In logs I've seen due to missing Supported NDK version it is skipping compression of library. On installing latest ndk version (ie 23) it was failing with toolkit error which got removed from ndk version 19. Any suggestion on supported ndk version on android studio so that library compression can be done which will reduce app size.

I am having same issue, Please help me

**Comment by matinzd** [comment]
> > In logs I've seen due to missing Supported NDK version it is skipping compression of library. On installing latest ndk version (ie 23) it was failing with toolkit error which got removed from ndk version 19. Any suggestion on supported ndk version on android studio so that library compression can be done which will reduce app size.
> 
> 
> 
> I am having same issue, Please help me

You can use ffmpeg kit for react native.
